net/safe-search-bing/Makefile

```diff
@@ -0,0 +1,60 @@
+#
+# Copyright (c) 2018 Gregory L. Dietsche (Gregory.Dietsche@cuw.edu)
+# This is free software, licensed under the MIT License
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=safe-search-bing
+PKG_VERSION:=1.0.0
+PKG_RELEASE:=4
```

Comment:
if this is a new package, should start at 1.  pkg_release is for new releases of the same version.

Reply:
Easy enough, I'll update my code, thanks! That's just left over from my testing.